src/components/token/utils.ts

```diff
@@ -0,0 +1,68 @@
+import { BitcoinNetwork } from '@rsksmart/rif-wallet-bitcoin'
+
+import { ITokenWithoutLogo } from 'src/redux/slices/balancesSlice/types'
+import { TokenSymbol } from 'src/screens/home/TokenImage'
+
+export type TokenOrBitcoinNetwork = ITokenWithoutLogo | BitcoinNetwork
```

Comment:
Is that related only to token as a component or it's overall token data? If it's the data then it's better be moved to `shared` folder